drools-ruleunits/drools-ruleunits-api/src/main/java/org/drools/ruleunits/api/DataStore.java

```diff
@@ -15,12 +15,20 @@
  */
 package org.drools.ruleunits.api;
 
+/**
+ * A {@link DataSource} of mutable data.
+ * @param <T> The type of objects managed by this DataSource.
+ */
 public interface DataStore<T> extends DataSource<T> {
 
+    /**
+     * Add an object to this DataStore.
+     * @return The {@link DataHandle} to further modified
```

Comment:
```suggestion
     * @return The {@link DataHandle} to be further modified
```
You wanted to write like this?